pom.xml

```diff
@@ -1097,6 +1144,45 @@
                             </archive>
                         </configuration>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-resources-plugin</artifactId>
+                        <executions>
+                            <execution>
+                            <id>copy-multirelease-sources-java10</id>
+                            <phase>prepare-package</phase>
+                            <goals>
+                                <goal>copy-resources</goal>
+                            </goals>
+                            <configuration>
+                            <outputDirectory>${java10.sources.metainf.location}/META-INF/versions/10</outputDirectory>
```

Comment:
Why can the sources not be included directly from their original locations?

Reply:
I gave it more try and the answer is yes, it is possible to copy them directly @dmlloyd .